OSDOCS#9497: editing canary rollout update page (#71004)
openshift/openshift-docs: modules/update-using-custom-machine-config-pools-mcp-remove.adoc

[id="update-using-custom-machine-config-pools-mcp-remove_{context}"]
= Moving a node to the original machine config pool

> After you have unpaused a custom machine config pool (MCP) and verified that the applications on the nodes associated with that MCP are working as expected, move the nodes back to their original MCP by removing the custom label you added to the nodes.

Author: Here the phrase "After you have unpaused a custom machine config pool (MCP) and verified that the applications on the nodes associated with that MCP are working as expected" is not a set of multiple instructions, instead it's a description of when you should perform the following procedure. It's saying "once you have performed X, move the nodes back to their original MCP by doing the procedure below."

I can try to shorten the phrase anyways, maybe by rewriting to the following:

```suggestion
After you have updated and verified applications on nodes with a custom machine config pool (MCP), move the nodes back to their original MCP by removing the custom label you added to the nodes.
```

Also, I think "move the nodes back to their original MCP by removing the custom label you added to the nodes" is already in active voice

[IMPORTANT]
====
A node must have a role to be properly functioning in the cluster.
====

> 

Author: I honestly go back and forth sometimes about whether there should be prerequisites for a procedure that include basically "accomplishment of the task in the previous section". Idk how well we adhere to our mod docs guidelines in reviews, but the [prereqs section](https://redhat-documentation.github.io/modular-docs/#prerequisites) states to "not list obvious prerequisites".

In this case, not only is the general workflow of described further up in the assembly, I would argue that any user following along in this assembly would see that there's no point in doing this last procedure unless they've accomplished the rest of the stuff preceding this. Let me know what you think, I can add a prereqs section if you think it would be helpful.

> 

Reviewer: No including it is fine with me.

If the node does not have a `worker` label or a label from an updated MCP, add the label.
////

> . For each node in a custom MCP, remove the custom label from the node by running the following command:

Author: That's not a bad idea, but per the [documentation guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command) that I'm supposed to follow, in most cases I should have "by running the following command" at the end of the instruction. I think that helps keep the step focused on the action you're performing rather than the fact that you're doing it by running a command.

+

> The MCO moves the nodes back to the original MCP and reconciles the node to the MCP configuration.

Author: That's a good catch, yes I think we should define it here. It's actually the only mention of the term in this section so I think I'll just write it out without an acronym:
```suggestion
The Machine Config Operator moves the nodes back to the original MCP and reconciles the node to the MCP configuration.
```

> . To ensure that node has been removed from the custom MCP, view the list of MCPs in the cluster and their current state by running the following command:

Author: Same response as above regarding the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command).

worker              rendered-worker-5ad4791166c468f3a35cd16e734c9028         True      False      False      3              3                   3                     0                      61m
----
+
Once the node is removed from the custom MCP and moved back to the original MCP, it can take several minutes to update the machine counts. In this example, one node was moved from the removed `workerpool-canary` MCP to the `worker` MCP.

> . Optional: Delete the custom MCP by running the following command:

Author: Same response regarding [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command).
